frontend/src/components/common/header/header.tsx

```diff
@@ -38,6 +38,14 @@ const Header: React.FC = () => {
                   Podcasts
                 </Link>
               </li>
+              <li className={styles.navigationItem}>
```

Comment:
will this item show for all users? we don't want it

Reply:
> will this item show for all users? we don't want it

only for authenticated users

Reply:
```suggestion
              {hasUser ? (
          <>
            <ul className={styles.navigation}>
              <li className={styles.navigationItem}>
                <Link to={AppRoute.ROOT} className={styles.link}>
                  Podcasts
                </Link>
              </li>
              <li className={styles.navigationItem}>
                <Link
                  to={`${AppRoute.PLAYLISTS_USERS}/${user?.id}`}
                  className={styles.link}
                >
                  My Playlists
                </Link>
              </li>
            </ul>